### DISCUSSION TITLE
Problem with GenericFigure

### DISCUSSION THREAD

**Discussion by nucleartux** [ueberdosis/tiptap#2189] [General Questions]
### What’s the bug you are facing?

Image disappears in GenericFigure when entering some text inside.

### How can we reproduce the bug on our side?

1. Open https://tiptap.dev/experiments/generic-figure
2. Click somewhere in "Image caption" text.
3. Click down arrow
4. Enter some text

### What did you expect to happen?

Image remains in place

### Anything to add? (optional)

Yes, I know what this is experiment. I have exactly same task. How we can achieve such behaviour with "fixed" nodes inside one another? 

### Did you update your dependencies?

- [X] Yes, I’ve updated my dependencies to use the latest version of all packages.

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by hanspagel** [comment]
Moving this to discussions, as stated on the experiments overview:

> Feel free to play around, but please, don’t open an issue for a bug you’ve found here or send pull requests. :-)
